scripts/lib/zulip_tools.py

```diff
@@ -355,23 +355,24 @@ def parse_lsb_release():
             vendor = 'RedHat'
             codename = 'rhel' + info[6][0]  # 7
         distro_info = dict(
-            DISTRIB_CODENAME=codename,
-            DISTRIB_ID=vendor,
+            VERSION_CODENAME=codename,
+            NAME=vendor,
             DISTRIB_FAMILY='redhat',
         )
         return distro_info
     try:
-        # For performance reasons, we read /etc/lsb-release directly,
+        # For performance reasons, we read /etc/os-release directly,
         # rather than using the lsb_release command; this saves ~50ms
         # in several places in provisioning and the installer
-        with open('/etc/lsb-release', 'r') as fp:
+        with open('/etc/os-release', 'r') as fp:
             data = [line.strip().split('=') for line in fp]
         for k, v in data:
-            if k not in ['DISTRIB_CODENAME', 'DISTRIB_ID']:
+            if k not in ['VERSION_CODENAME', 'NAME']:
                 # We only return to the caller the values that we get
                 # from lsb_release in the exception code path.
                 continue
-            distro_info[k] = v
+            # The strip() removes superfluous '"'
+            distro_info[k] = v.strip('"')
         distro_info['DISTRIB_FAMILY'] = 'debian'
```

Comment:
We should at some point replace this `DISTRIB_FAMILY` hack with the `/etc/os-release` `ID_LIKE` field (`if 'debian' in [distro_info['ID'], *distro_info['ID_LIKE'].split()]`).

Reply:
#13070